### PR TITLE
@W-16470063 - Update to-publish-assets-maven.adoc

### PR DESCRIPTION
**Pull request by marianomerlo**
Adding an example about how to publish a parent pom and a dependent project both to Exchange.

# Writer's Quality Checklist

Before merging your PR, did you:

- [ ] Run spell checker
- [ ] Run link checker to check for broken xrefs
- [ ] Check for orphan files
- [ ] Perform a local build and do a final visual check of your content, including checking for:
  - Broken images
  - Dead links
  - Correct rendering of partials if they are used in your content
  - Formatting issues, such as:
    - Misnumbered ordered lists (steps) or incorrectly nested unordered lists
    - Messed up tables
    - Proper indentation
    - Correct header levels
- [ ] Receive final review and signoff from:
  - Technical SME
  - Product Manager
  - Editor or peer reviewer
  - Reporter, if this content is in response to a reported issue (internal or external feedback)
- [ ] If applicable, verify that the software actually got released

### CODE REVIEW

File: modules/ROOT/pages/to-publish-assets-maven.adoc [hunk 1]
```diff
@@ -346,6 +346,43 @@ This example shows the `build` section of `pom.xml`:
 </build>
 ----
 
+== Publish a parent pom project and a parent pom dependant project to Exchange
```
Comment (JennyHajee) [inline modules/ROOT/pages/to-publish-assets-maven.adoc]:
```suggestion
== Publish a Parent POM Project and a Parent POM Dependant Project to Exchange
```

File: modules/ROOT/pages/to-publish-assets-maven.adoc [hunk 2]
```diff
@@ -346,6 +346,43 @@ This example shows the `build` section of `pom.xml`:
...
+
+Parent poms are published to Exchange as custom assets. See previous section in order to publish a custom asset.
```
Comment (JennyHajee) [inline modules/ROOT/pages/to-publish-assets-maven.adoc]:
```suggestion
Parent POMs are published to Exchange as custom assets. See the previous section to publish a custom asset.
```

File: modules/ROOT/pages/to-publish-assets-maven.adoc [hunk 3]
```diff
@@ -346,6 +346,43 @@ This example shows the `build` section of `pom.xml`:
...
+If you want to publish a maven project, like a mule application, using the Mule Maven Plugin, and your maven project depends on a Parent Pom that has already included the exchange-mule-maven-plugin, you need to https://maven.apache.org/guides/mini/guide-configuring-plugins.html#Using_the_.3Cinherited.3E_Tag_In_Build_Plugins[prevent the inheritance]:
```
Comment (JennyHajee) [inline modules/ROOT/pages/to-publish-assets-maven.adoc:352]:
```suggestion
If you want to publish a Maven project, like a Mule application, using the Mule Maven plugin, and your Maven project depends on a parent that is already included in the exchange-mule-maven-plugin, you must https://maven.apache.org/guides/mini/guide-configuring-plugins.html#Using_the_.3Cinherited.3E_Tag_In_Build_Plugins[break the inheritance]:
```

Comment (marianomerlo) [reply modules/ROOT/pages/to-publish-assets-maven.adoc:352]: This is not why i actually mean. 
The desired output will be something like: "if your maven project uses the mule-maven-plugin, and depends on a parent on that already includes the exchange-mule-maven-plugin, you must break the inheritance"

**Review by JennyHajee — APPROVED** [review]

**Pull request merged**